packages/components/src/Popover/Layout/PopoverFooter/PopoverFooter.story.tsx

```diff
@@ -25,33 +25,26 @@
  */
 
 import React from 'react'
+import { ButtonTransparent } from '../../../Button/ButtonTransparent'
 import { PopoverFooter } from './PopoverFooter'
 
 export default {
   component: PopoverFooter,
   title: 'PopoverFooter',
 }
 
-export const Basic = () => <PopoverFooter>Footer Text</PopoverFooter>
-Basic.parameters = {
-  storyshots: {
-    disable: true,
-  },
-}
+export const Basic = () => <PopoverFooter />
 
-export const CloseStringValue = () => (
-  <PopoverFooter close="Footer">Footer Text</PopoverFooter>
+export const FooterWithChildren = () => (
+  <PopoverFooter>
+    <ButtonTransparent color="neutral" size="small">
+      cancel
```

Comment:
Would be nice to capitalize this so it matches `done`

```suggestion
      Cancel
```